ouf.lua

```diff
@@ -567,10 +567,9 @@ end
 function oUF:SpawnNamePlates(namePrefix, nameplateCallback, nameplateCVars)
 	argcheck(nameplateCallback, 3, 'function', 'nil')
 	argcheck(nameplateCVars, 4, 'table', 'nil')
-
-	local style = style
 	if(not style) then return error('Unable to create frame. No styles have been registered.') end
 
+	local style = style
```

Comment:
Because next `oUF:SetActiveStyle` call will overwrite style that's activated for nameplates before calling `oUF:SpawnNamePlates`. We have to cache it, cuz nameplates don't spawn right away.